src/modules/billing/mappers/api/invoice.js

```diff
@@ -9,6 +9,19 @@ const containsWaterUndertaker = invoiceLicences => {
     .includes(true);
 };
 
+const nullToEmpty = (val) => val === null ? '' : val;
+const buildName = (name) => `${nullToEmpty(name.salutation)} ${nullToEmpty(name.firstName)} ${nullToEmpty(name.middleInitials)} ${nullToEmpty(name.lastName)}`;
```

Comment:
This can generate spaces where they are not needed. 

But this is managed during the page render with `title case`